simulator/util/src/main/java/com/powsybl/incubator/simulator/util/EquationSystemBusFeedersResult.java

```diff
@@ -48,29 +50,26 @@ public void updateContributions() {
         double bSum = 0.;
         double gSum = 0.;
 
-        for (ShortCircuitResultFeeder feeder : busFeedersResults) {
+        for (EquationSystemResultFeeder feeder : busFeedersResults) {
             bSum = bSum + feeder.getB();
             gSum = gSum + feeder.getG();
         }
 
-        for (ShortCircuitResultFeeder feederResult : busFeedersResults) {
-            double epsilon = 0.000001;
-            if (Math.abs(gSum) > epsilon || Math.abs(bSum) > epsilon) {
-                double det = 1 / (gSum * gSum + bSum * bSum);
+        if (Math.abs(gSum) > EPSILON || Math.abs(bSum) > EPSILON) {
+            double det = 1 / (gSum * gSum + bSum * bSum);
```

Comment:
Probably a Sonar false positive, because of the previous check it cannot be zero.

Reply:
Agree